src/Twig/Extension/SonataAdminExtension.php

```diff
@@ -245,6 +247,7 @@ public function getValueFromFieldDescription(
         FieldDescriptionInterface $fieldDescription,
         array $params = []
     ) {
+        // NEXT_MAJOR: remove this check
```

Comment:
```suggestion
        if ([] !== $params) {
            @trigger_error('Passing a third parameter is deprecated since sonata-project/admin-bundle 3.x and will be ignored in 4.0.', E_USER_DEPRECATED)
        }

        // NEXT_MAJOR: remove this check
```